pliops/producer.cc

```diff
@@ -96,7 +96,9 @@ void Producer::ReaderThread(uint32_t iterator_parallelism_factor, uint32_t threa
 
     if (!EnqueueTimed(message_queue_, {std::string(key.data(), key.size()), std::string(value.data(), value.size())}, kill_, ops_timeout_msec_)) {
       // We must release the iterator here, otherwise DB::Close will crash
-      status = iterator->Close();
+#ifdef XDPROCKS
+      iterator->Close();
+#endif
```

Comment:
why remove the status and not add it to the error message?

Reply:
this API only exists in XdpRocks